tests/e2e/features/display_search.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Display search functionality
+    @wip
+    Scenario: As a user I want to display search results
+        When I visit the "homepage" page
+
+        Then I scroll to the "search_input" element
+        When I fill the "search_input" field with "consequatur"
+        And I click the "search_button" element
+
```

Comment:
add "search" page

Reply:
What do you mean?

Reply:
Add new PageObject and then check if you are on right page

Reply:
How? :-)